### DISCUSSION TITLE
Dúvida sobre Grupo de discussão

### DISCUSSION THREAD

**Discussion by charleston10** [raulfdm/taco-api#73] [Q&A]
Olá! Eu estou interessado em participar mais ativamente no desenvolvimento da API e gostaria de saber se existe algum grupo de discussão, além do GitHub, onde a comunidade se reúne. Estou procurando por espaços como Slack, Discord, ou qualquer outra plataforma de comunicação, onde possamos trocar ideias, solucionar dúvidas e compartilhar progressos relacionados ao desenvolvimento da API. Isso existe? Se sim, poderiam me informar como faço para me juntar? Obrigado!

**Comment by raulfdm** [comment]
E aí @charleston10 , beleza?

Não existe mano. Quando criei o projeto fiz sozinho. As pessoas foram fazendo fork e criando seus projetos em cima dele, então essa comunidade não existe.
